From eab19607a836f1b632e9a78630279ab483ceb14e Mon Sep 17 00:00:00 2001
From: Michael Healy <horsemanoffaith@gmail.com>
Date: Sat, 23 Jan 09:00:00 -0800
Subject: [PATCH 1/8] Disable tests

--- a/CMakeLists.txt	2016-01-23 04:48:05.721175968 -0800
+++ b/CMakeLists.txt	2016-01-23 04:48:31.691175732 -0800
@@ -289,53 +289,6 @@
 add_subdirectory(gnome)
 
 if (ENABLE_UNIT_TESTS)
-  # Check for google test and build it locally
-  set (
-    GMOCK_ROOT_DIR
-    "/usr/src/gmock"
-    CACHE
-    PATH
-    "Path to Google mock srcs"
-  )
-
-  set (
-    GMOCK_INCLUDE_DIR
-    "/usr/include/gmock/include"
-    CACHE
-    PATH
-    "Path to Google mock include"
-  )
-
-  set(
-    GTEST_ROOT_DIR
-    "${GMOCK_ROOT_DIR}/gtest" # Default value, adjustable by user with e.g., ccmake
-    CACHE
-    PATH
-    "Path to Google test srcs"
-  )
-
-  set(
-    GTEST_INCLUDE_DIR
-    "${GMOCK_ROOT_DIR}/gtest/include"
-    CACHE
-    PATH
-    "Path to Google tests include"
-  )
-
-  if (GTEST_INCLUDE_DIR AND GMOCK_INCLUDE_DIR)
-    include_directories (
-      ${GMOCK_INCLUDE_DIR}
-      ${GTEST_INCLUDE_DIR}
-    )
-    add_subdirectory(
-      ${GMOCK_ROOT_DIR}
-      "${CMAKE_CURRENT_BINARY_DIR}/gmock"
-    )
-  elseif (GTEST_INCLUDE_DIR AND NOT GMOCK_INCLUDE_DIR)
-    include_directories (
-      ${GTEST_INCLUDE_DIR}
-    )
-  endif(GTEST_INCLUDE_DIR AND GMOCK_INCLUDE_DIR)
 
   add_subdirectory(tests)




