From 2fd0578939f7ca9387c49012ce39e1a2fcc5f949 Mon Sep 17 00:00:00 2001
From: Xiao-Long Chen <chenxiaolong@cxl.epac.to>
Date: Fri, 4 Apr 2014 16:03:27 -0400
Subject: [PATCH] Remove Ubuntu One support: service is dead

---
 Makefile.am                              |   5 +-
 po/POTFILES.skip                         |   3 -


diff --git a/Makefile.am b/Makefile.am
index 0aca69a..1c9260e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -1,12 +1,11 @@
 SUBDIRS = src data po tests
 
 #
-# Install the music.lens and musicstore.scope files
+# Install the music.lens file
 #
 scope_in_files = \
   banshee.scope.in \
-  rhythmbox.scope.in \
-  musicstore.scope.in
+  rhythmbox.scope.in
 scopedir = $(SCOPESDIR)/music
 scope_DATA = $(scope_in_files:.scope.in=.scope)
 
diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index d7c2320..32e2313 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -4,6 +4,3 @@ src/utils.c
 src/simple-scope.c
 src/banshee-scope.c
 src/rhythmbox-scope.c
-src/musicstore-scope.c
-src/ubuntuone-webservices.c
-tests/unit/ubuntuone-webservices.c
-- 
2.1.2
